support/openharmony/entry/src/main/ets/pages/Index.ets

```diff
@@ -133,6 +158,7 @@ struct Index {
                 this.urlToLoad = new_url
               })
               this.xComponentContext.registerPromptToastCallback(prompt_toast)
+              this.setupAvoidAreaChangeListener()
```

Comment:
I'm wondering a bit what does it mean in ArkTS if we call an `async` function here. When can we assume that the function completed?